shadowapi/src/main/java/org/robolectric/util/ReflectionHelpers.java

```diff
@@ -316,12 +316,12 @@ public static <R> R callInstanceMethod(
   /**
    * Helper method for calling a static method using a class from a custom class loader
    *
-   * @param classLoader
-   * @param fullyQualifiedClassName
-   * @param methodName
-   * @param classParameters
-   * @param <R>
-   * @return
+   * @param classLoader The class loader of calling method
+   * @param fullyQualifiedClassName The name of the class prefixed with the package
+   * @param methodName The method name
+   * @param classParameters Parameters of class which contains the method
+   * @param <R> Return type of the method
+   * @return Return the value of the method
    */
   public static <R> R callStaticMethod(
```

Comment:
Added the description according to my own understanding by reading the neighboring code. Please feel free to leave your suggestion. Thanks :)

Reply:
classLoader  The ClassLoader used to load calss.
fullQulifiedClassName The full qualified class name with package name of the ClassLoader will load. See https://docs.oracle.com/javase/specs/jls/se11/html/jls-6.html#jls-6.7 for details.
methodName The method name will be called.
classParameters The input parameters will be used for method calling.

cc @hoisie

Reply:
Changed things according to the suggestions

Reply:
@hoisie Could you help recheck the grammar? Thanks

Reply:
I use Grammarly to check the grammar, this is mainly suggested
- @param fullyQualifiedClassName I've checked grammar The fully qualified class name with a package name of the ClassLoader will load
- @param methodName The method name will be called
- @return The return the value of the method

Reply:
@hoisie any update?